pkg/kubectl/stop.go

```diff
@@ -272,6 +280,54 @@ func (reaper *JobReaper) Stop(namespace, name string, timeout time.Duration, gra
 	return nil
 }
 
+func (reaper *DeploymentReaper) Stop(namespace, name string, timeout time.Duration, gracePeriod *api.DeleteOptions) error {
+	deployments := reaper.Extensions().Deployments(namespace)
+	rcs := reaper.ReplicationControllers(namespace)
+	scaler, err := ScalerFor(extensions.Kind("Deployment"), *reaper)
```

Comment:
I don't think you need to use a scaler for deployments here. Just get all the rcs of the deployment and reap them. Delete the deployment first so that the deployment controller won't recreate it in the meantime. See https://github.com/openshift/origin/blob/master/pkg/deploy/reaper/reaper.go for our existing deployment reaper.